Podcast/UserSeriesSupplierView.swift

```diff
@@ -125,9 +125,9 @@ class UserSeriesSupplierView: UIView {
             }
             
             if users.count > 3 {
-                contextString.append(NSAttributedString(string: ", and " + String(users.count - 3) + " others recommended this \(recommendationType)"))
+                contextString.append(NSAttributedString(string: ", and " + String(users.count - 3) + " others subscribed to this \(recommendationType)"))
```

Comment:
you'll probably want to merge with master because I remember touching this text with my recent PR

Oh also we shouldn't be changing the text to subscribe for an episode but only for a series! (i.e. You "recommend" an episode and "subscribe" to a series)

In the image below the episode cell should still say "Jack Thompson recommended this episode"

<img width="258" alt="screen shot 2018-02-25 at 11 16 56 pm" src="https://user-images.githubusercontent.com/14966713/36653411-0961abda-1a83-11e8-8403-c6b9e35e0dce.png">

Reply:
how do i merge lol

Reply:
`git fetch`, `git merge origin master`, ... hopefully you won't have conflicts. It should auto-merge. If not! do `git abort` and I can show you how to deal with merge conflicts in person